lib/components/af.js

```diff
@@ -378,7 +378,7 @@ af.zclFunctional = function (srcEp, dstEp, cId, cmd, zclData, cfg, callback) {
 /*************************************************************************************************/
 /*** ZCL Cluster and Attribute Requests                                                        ***/
 /*************************************************************************************************/
-af.zclClustersReq = function (dstEp, callback) {    // callback(err, clusters)
+af.zclClustersReq = function (dstEp, eventEmitter, callback) {    // callback(err, clusters)
```

Comment:
Added a check, if eventEmitter is a function, consider it the callback and do not emit events.